protos/tunneloffload.proto

```diff
@@ -0,0 +1,352 @@
+// Add licensing
```

Comment:
refer to DriveNets, VMWare and AT&T in the copyright line of the Apacke 2.0 license